lib/rules/jsx-no-literals.js

```diff
@@ -44,6 +44,7 @@ module.exports = {
 
     function getValidation(node) {
       const standard = !/^[\s]+$/.test(node.value) &&
+          typeof node.value === 'string' &&
```

Comment:
```javascript
<Foo
  a={true}
  b={1}
  c={null} />
```
all of those props get hit, so typeof could be `boolean`, `number`, and when null it's an `object`